manifests/DEPRECATED.md

```diff
@@ -0,0 +1,5 @@
+# Old manifests format
```

Comment:
I believe our only restriction is us... on the testing side.  we currently build the bundle in CI

Reply:
Right, I kept this to avoid breaking CI. Once this is merged we can switch CI also to the operator SDK method.